lib/widgets/adaptive_flat_button.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+
+class AdaptiveFlatButton extends StatelessWidget {
+  final String text;
+  final Function handler;
+
+  AdaptiveFlatButton(this.text, this.handler);
+
+  @override
+  Widget build(BuildContext context) {
+    return Platform.isIOS
+                        ? CupertinoButton(
+                            child: Text(
+                              text,
+                              style: TextStyle(fontWeight: FontWeight.bold),
+                            ),
+                            onPressed: handler,
+                          )
+                        : FlatButton(
+                            textColor: Theme.of(context).primaryColor,
+                            child: Text(
+                              text,
+                              style: TextStyle(fontWeight: FontWeight.bold),
+                            ),
+                            onPressed: handler,
+                          );
+  }
+}
```

Comment:
@MapleSyyrup this is still wrong. add a new line that does not have this closing bracket

Reply:
> @MapleSyyrup this is still wrong. add a new line that does not have this closing bracket

@Nei1eveN a blank line was added
![image](https://user-images.githubusercontent.com/74462760/100755309-03f9bf00-3427-11eb-9b7d-c1349acdd527.png)

Reply:
@MapleSyyrup not the way I see it
![image](https://user-images.githubusercontent.com/41381713/100755974-b6318680-3427-11eb-9aa4-c4ed85c3c644.png)

Reply:
@MapleSyyrup make sure to apply a blank line on your new commit